src/components/remotecontrol/remotecontrol.js

```diff
@@ -1,939 +1,954 @@
+            events.on(player, 'playlistitemremove', onPlaylistItemRemoved);
+            events.on(player, 'playlistitemmove', onPlaylistUpdate);
+            events.on(player, 'playlistitemadd', onPlaylistUpdate);
+            events.on(player, 'playbackstop', onPlaybackStopped);
+            events.on(player, 'volumechange', onVolumeChanged);
+            events.on(player, 'pause', onPlayPauseStateChanged);
+            events.on(player, 'unpause', onPlayPauseStateChanged);
+            events.on(player, 'timeupdate', onTimeUpdate);
+            const playerInfo = playbackManager.getPlayerInfo();
+            const supportedCommands = playerInfo.supportedCommands;
+            currentPlayerSupportedCommands = supportedCommands;
+            updateSupportedCommands(context, supportedCommands);
         }
+    }
 
-        function onBtnCommandClick() {
-            if (currentPlayer) {
-                if (this.classList.contains('repeatToggleButton')) {
-                    toggleRepeat();
-                } else {
-                    playbackManager.sendCommand({
-                        Name: this.getAttribute('data-command')
-                    }, currentPlayer);
-                }
+    function onBtnCommandClick() {
+        if (currentPlayer) {
+            if (this.classList.contains('repeatToggleButton')) {
+                toggleRepeat();
+            } else {
+                playbackManager.sendCommand({
+                    Name: this.getAttribute('data-command')
+                }, currentPlayer);
             }
         }
+    }
 
-        function getSaveablePlaylistItems() {
-            return getPlaylistItems(currentPlayer).then(function (items) {
-                return items.filter(function (i) {
-                    return i.Id && i.ServerId;
-                });
+    function getSaveablePlaylistItems() {
+        return getPlaylistItems(currentPlayer).then(function (items) {
+            return items.filter(function (i) {
+                return i.Id && i.ServerId;
             });
-        }
+        });
+    }
 
-        function savePlaylist() {
-            require(['playlistEditor'], function (playlistEditor) {
-                getSaveablePlaylistItems().then(function (items) {
-                    var serverId = items.length ? items[0].ServerId : ApiClient.serverId();
-                    new playlistEditor.showEditor({
-                        items: items.map(function (i) {
-                            return i.Id;
-                        }),
-                        serverId: serverId,
-                        enableAddToPlayQueue: false,
-                        defaultValue: 'new'
-                    });
+    function savePlaylist() {
+        import('playlistEditor').then(({ default: playlistEditor }) => {
+            getSaveablePlaylistItems().then(function (items) {
+                const serverId = items.length ? items[0].ServerId : ApiClient.serverId();
+                new playlistEditor.showEditor({
```

Comment:
`showEditor` is `default`
```suggestion
                new playlistEditor({
```